src/Groups/cosets.jl

```diff
@@ -43,6 +64,11 @@ julia> g = perm(G,[3,4,1,5,2])
 
 julia> H = symmetric_group(3)
 Permutation group of degree 3 and order 6
+
+julia> right_coset(H, g)
+Right coset of permutation group of degree 3 and order 6
+  with representative (1,3)(2,4,5)
+  in permutation group of degree 5 and order 120
```

Comment:
I think this will have to be updated to adjust for the other PR which causes symmetric groups to be printed as `Sym(n)` ?

Reply:
Yes. (It was not clear which pull request would get merged first.)